pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -664,7 +777,7 @@ class PCPhase(Operation):
 
     def generator(self) -> "qml.Hermitian":
```

Comment:
since you link to the generator method might be worth adding a docstring. Currently it is taking the doc string of the parent class (I guess?)
![image](https://github.com/user-attachments/assets/f9e3e6ca-e545-4cfd-b8cc-e173d4ecac82)

Reply:
Very good point!

Reply:
Done :heavy_check_mark: